__tests__/app.test.js

```diff
@@ -140,12 +140,49 @@ describe("Task 5 - GET /api/articles", () => {
 });
 
 describe("Task 6 - GET /api/articles/:article_id/comments", () => {
-  test("should test this function", () => {
+  test("should return an array of objects with the appropriate properties", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
       .then((response) => {
-        expect(response).toBe("broken on purpose - finished for day");
+        const comments = response.body.comments;
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("should return an array that is sorted by the newest article first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const comments = response.body.comments;
+        expect(comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("should return a 404 error if no comments exist for the article", () => {
+    return request(app)
+      .get("/api/articles/10/comments")
```

Comment:
if an article has not been commented on yet - do we expect to receive an error at this point?